lpr.WebAPI/Middleware/JWTMiddleware.cs

```diff
@@ -0,0 +1,37 @@
+﻿using lpr.Common.Interfaces.Contexts;
+using lpr.Common.Interfaces.Services;
+using System.Security.Claims;
+
+namespace lpr.WebAPI.Middleware
+{
+    public class JWTMiddleware
+    {
+        private readonly RequestDelegate _next;
+
+        public JWTMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+
+        public async Task Invoke(HttpContext context, ILprDbContext dbContext, IJWTService jwtService)
+        {
+            string token = context.Request.Headers["X-JWT-Token"];
+            try
+            {
+                if (jwtService.IsTokenValid(token))
+                {
+                    IEnumerable<Claim> claims = jwtService.GetTokenClaims(token);
+                    string githubId = claims.FirstOrDefault(e => e.Type.Equals(ClaimTypes.Authentication)).Value;
+
+                    context.Items["GitHubAccessToken"] = githubId;
```

Comment:
This should account id instead of GitHub id

Reply:
Correct this is a bug.